### DISCUSSION TITLE
How to set the file name when creating new note from template?

### DISCUSSION THREAD

**Discussion by LeducH** [SilentVoid13/Templater#277] [Help] [answered]
I want to set the file name to `prefix-YYYY-MM-DD` then I create a new note from template. 

How can this be done?

**Comment by SilentVoid13** [comment, marked as answer]
Hey @LeducH, you can add the following command inside your template:

```js
<% await tp.file.rename("prefix-" + tp.date.now()) %>
```

**Reply by LeducH** [reply]
sweet thank you.
